aerleon/lib/fqdn.py

```diff
@@ -0,0 +1,19 @@
+class FQDN:
```

Comment:
It looks like FQDN could just be a Python dataclass. The default __eq__ method generated by the dataclass decorator will be identical to your implementation here, so you can remove it.

```python
from dataclasses import dataclass

@dataclass
class FQDN:
    """Docstring pls"""
    fqdn: str
    text: float
    token: token
    parent_token: str
    
    def __init__(self, fqdn: str, token: str, comment: str = ''):
        self.fqdn = fqdn
        self.text = comment
        self.token = token
        self.parent_token = token
```


https://docs.python.org/3/library/dataclasses.html

Reply:
Done